lib/v1.recommended.yaml

```diff
@@ -124,6 +124,9 @@ analyzer:
     use_function_type_syntax_for_parameters: warning
     use_rethrow_when_possible: warning
 
+    # Ignore error:
+    uri_has_not_been_generated: ignore
```

Comment:
I think it'd be helpful to add the https://github.com/Workiva/over_react/blob/new_boilerplate_wip/doc/new_boilerplate_migration.md#ignore-ungenerated-warnings-project-wide link here in the comment so that it's clear why we're ignoring `uri_has_not_been_generated` at this level